src/main/java/net/snowflake/client/jdbc/SnowflakeFileTransferAgent.java

```diff
@@ -2976,7 +2976,7 @@ public CommandType getCommandType() {
   /*
    * Handles an InvalidKeyException which indicates that the JCE component
    * is not installed properly
-   * @param operation a string indicating the the operation type, e.g. upload/download
+   * @param operation a string indicating the operation type, e.g. upload/download
```

Comment:
I'm still experimenting w/ duplicate word detection/correction. I'm pretty sure this is right (there's one doubled word that I didn't change -- iirc it was in an `@ ... ` doc block)